src/comps/OrderCompleteDialog.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import {
+    Dialog,
+    DialogTitle,
+    DialogActions,
+    Button,
+} from '@mui/material';
+
+interface OrderCompleteProps {
+    open: boolean;
+    onCloseClick: () => void;
+}
+
+const OrderCompleteDialog: React.FC<OrderCompleteProps> = ({open, onCloseClick}) => {
```

Comment:
```suggestion
const OrderCompleteDialog = (props: OrderCompleteProps) => {
```
in globus we use a different syntax for this